src/syrupy/assertion.py

```diff
@@ -31,6 +42,7 @@ def __init__(
         self._serializer_class = serializer_class
         self._test_location = test_location
         self._executions = 0
+        self._execution_results: Dict[int, AssertionResult] = {}
```

Comment:
Perhaps we should stick to forward ref style, i.e. quotes, for all class types? For consistency

Reply:
Missed it